src/main/java/network/brightspots/rcv/RawContestConfig.java

```diff
@@ -245,6 +266,56 @@ public boolean isExcluded() {
     public void setExcluded(boolean excluded) {
       this.excluded = excluded;
     }
+
+    /**
+     * A stream of all aliases (which is guaranteed to be unique) and the candidate name
+     * (which is not guaranteed to be unique, i.e. it may exist in the list twice)
```

Comment:
why might the name be present twice?

Reply:
It's valid for a candidate name to be equal to a candidate code, and therefore for a candidate name to be equal to an alias.